src/rsz/README.md

```diff
@@ -92,26 +92,24 @@ estimate_parasitics
 | ----- | ----- |
 | `-placement` or `-global_routing` | Either of these flags must be set. Parasitics are estimated based after placement stage versus after global routing stage. |
 
-### Set Don't Use
+### Set Dont Use
```

Comment:
why remove the '?

Reply:
It is used in regex parsing as manpage filenames, so as to match lowercase Tcl commands `set_dont_use`. I know it's not perfect, but will work on it improving this after the tests are done